pyvista/plotting/plotter.py

```diff
@@ -2058,11 +2058,20 @@ def reset_key_events(self):
         b_left_down_callback = lambda: self.iren.add_observer(
             'LeftButtonPressEvent', self.left_button_down
         )
+
+        def zoom_in():  # numpydoc ignore=GL08
+            self.camera.Zoom(1.05)
```

Comment:
We wrapped `Zoom` method by `zoom`. Let's use it!
```suggestion
            self.camera.zoom(1.05)
```
https://github.com/pyvista/pyvista/blob/a0691dd6687774247ff6e806e801a48c37085023/pyvista/plotting/camera.py#L410-L459